### PR TITLE
update website publishing info

### PR DESCRIPTION
**Pull request by 317brian**

### CODE REVIEW

File: distribution/asf-release-process-guide.md [hunk 1]
```diff
@@ -623,36 +623,41 @@ http://www.apache.org/legal/release-policy.html#release-announcements
...
+   # Include `--skip-install` if you already have Docusaurus 2 installed in druid-website-src. 
+   # The script assumes you use `npm`. If you use `yarn`, include `--yarn`.
 
-6. Make a PR to the src repo (https://github.com/apache/druid-website-src) for the master branch.
+   python do_all_things.py -v VERSION --source /my/path/to/apache/druid
+   ```
    
-7. Make a PR to the website repo (https://github.com/apache/druid-website) for the `asf-site` branch. Once the website PR is merged, https://druid.apache.org/ will be updated immediately.
+
+5. Add the newly built files to the PR to the src repo (https://github.com/apache/druid-website-src) for the release branch. In the changed files, you should see the following:
```
Comment (317brian) [inline distribution/asf-release-process-guide.md:650]:
```suggestion
5. Add the files to a PR to the src repo (https://github.com/apache/druid-website-src) for the release branch you just created. In the changed files, you should see the following:
```

File: distribution/asf-release-process-guide.md [hunk 2]
```diff
@@ -370,15 +370,15 @@ $ svn commit -m 'add 0.17.0-rc3 artifacts'
...
 For more thorough instructions and a description of what the `do_all_things` script does, see the [`druid-website-src` README](https://github.com/apache/druid-website-src)
 
 1. Pull https://github.com/apache/druid-website and https://github.com/apache/druid-website-src. These repositories should be in the same directory as your Druid repository that should have the release tag checked out.
 
-2. From `druid-website`, checkout branch `asf-staging`.
+2. From `druid-website`, checkout a staging branch based off of the `asf-staging` branch.
 
-3. From `druid-website-src`, create a release branch from `master`, such as `27.0.0-docs`.
+3. From `druid-website-src`, create a release branch from `master`, such as `27.0.0-staging`.
    1. Update the version list in `static/js/version`.js with the version you're releasing and the release date. The highest release version goes in position 0.
```
Comment (vtlim) [inline distribution/asf-release-process-guide.md:382]:
```suggestion
   1. Update the version list in `static/js/version.js` with the version you're releasing and the release date. The highest release version goes in position 0. 
```

File: distribution/asf-release-process-guide.md [hunk 3]
```diff
@@ -623,36 +623,41 @@ http://www.apache.org/legal/release-policy.html#release-announcements
...
+   ```
    
-7. Make a PR to the website repo (https://github.com/apache/druid-website) for the `asf-site` branch. Once the website PR is merged, https://druid.apache.org/ will be updated immediately.
+
+5. Add the files to a PR to the src repo (https://github.com/apache/druid-website-src) for the release branch you just created. In the changed files, you should see the following:
+  - In `published_versions` directory: HTML files for `docs/VERSION` , `docs/latest`, and assorted HTML and non-HTML files. 
+  - In the `docs` directory at the root of the repo, the new Markdown files.
+    
+    All these files should be part of your PR to `druid-website-src`.
+   <br />
```
Comment (vtlim) [inline distribution/asf-release-process-guide.md:655]:
```suggestion
    All these files should be part of your PR to `druid-website-src`.  
```
Markdown isn't being rendered properly in the "Verify the site..." line

image

File: distribution/asf-release-process-guide.md [hunk 4]
```diff
@@ -397,7 +397,7 @@ For more thorough instructions and a description of what the `do_all_things` scr
     All these files should be part of your PR to `druid-website-src`.
    <br />
```
Comment (vtlim) [inline distribution/asf-release-process-guide.md:398]:
```suggestion
    All these files should be part of your PR to `druid-website-src`.  
```

**Review by vtlim — APPROVED** [review]
Content update looks good

**Pull request closed without merging**